src/pages/registration/files_and_license_tab/DownloadFileButton.tsx

```diff
@@ -11,9 +11,10 @@ import { openFileInNewTab } from '../../../utils/registration-helpers';
 
 interface DownloadFileButtonProps {
   file: AssociatedFile;
+  greyTones?: boolean;
```

Comment:
Using name greyTones instead of "disabled" because when in published-array it is still possible to click and use, but will have colors that match the rest of the array (i.e. greyTones)